Sources/FioriSwiftUICore/_FioriStyles/SideBarStyle.fiori.swift

```diff
@@ -15,24 +15,35 @@ import SwiftUI
 // Base Layout style
 public struct SideBarBaseStyle: SideBarStyle {
     @Environment(\.editMode) private var editMode
+    @Environment(\.sidebarLeadingPadding) private var leadingPadding
+    @Environment(\.sidebarTrailingPadding) private var trailingPadding
+    @Environment(\.sidebarWidth) private var sidebarWidth
+    @Environment(\.isSidebarAutoWidth) private var isAutoWidth
     @EnvironmentObject private var modelObject: SideBarModelObject
     @State private var collapsedSections: [UUID] = [] // To keep the collapsed section ID
     
     public func makeBody(_ configuration: SideBarConfiguration) -> some View {
         Group {
+            // See issue HCPSDKFIORIUIKIT-3047, With the iOS 26 Liquid Glass style, the Sidebar does not display fully vertically on the iPad in Portrait mode.
+            // It seems that the primary column's width in NavigationSplitView or UINavigationSplitViewControoler has changed, making the original fixed width of the sidebar (288 + 16 + 16) unusable.
+            // To resolve this issue, we need to implement a flexible width that fits the available content space of its parent.
+            
+            // Additionally, we also expose some environment keys, such as IsSidebarAutoWidth, SidebarWidth, SidebarLeadingPadding, and SidebarTrailingPadding, to allow consumers to customize the sidebar's width for different use cases.
```

Comment:
```suggestion
            // Additionally, we also expose some environment keys, such as isSidebarAutoWidth, sidebarWidth, sidebarLeadingPadding, and sidebarTrailingPadding, to allow consumers to customize the sidebar's width for different use cases.
```
Also update other places where these names are used.
This lowercase convention comes from Apple. (https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html)

Reply:
Fixed